src/datasource/requestLooper.ts

```diff
@@ -80,7 +81,34 @@ export function getRequestLooper<T extends DataQuery = DataQuery>(
             .subscribe(observer);
           nextQueries = undefined;
         } else {
-          subscriber.complete();
+          // check for queries that are opted for streaming
+          const targets = req.targets.filter((t) => {
+            return t.queryType === TwinMakerQueryType.EntityHistory && t.isStreaming;
+          });
+          if (targets.length === 0) {
+            subscriber.complete();
+          } else {
+            tracker.fetchEndTime = undefined;
+            tracker.fetchStartTime = Date.now();
+            setTimeout(() => {
+              subscription = options
+                .query({
+                  ...req,
+                  targets: targets,
+                  requestId: `${req.requestId}.${++count}`,
+                  startTime: tracker.fetchStartTime ?? 0,
+                  range: {
+                    from: req.range.to,
+                    to: dateTime(Date.now()),
+                    raw: {
+                      from: req.range.to,
+                      to: dateTime(Date.now()),
+                    },
+                  },
+                })
+                .subscribe(observer);
+            }, req.intervalMs);
```

Comment:
@hwandersman Currently we are polling this new data every intervalMs, which is dependent on the time range that user sets. Any thoughts on this interval? 
![Screen Shot 2022-07-06 at 11 51 31 AM](https://user-images.githubusercontent.com/17167829/177592287-c761ead3-73a2-4448-8711-aaa3243a3efb.png)